app-tracking-protection/vpn-impl/src/test/java/com/duckduckgo/mobile/android/vpn/ui/tracker_activity/DeviceShieldActivityFeedViewModelTest.kt

```diff
@@ -244,7 +244,10 @@ class DeviceShieldActivityFeedViewModelTest {
     }
 
     companion object {
-        private val timeWindow = TimeWindow(1, DAYS)
+        private val timeWindow = TimeWindow(
+            config.timeWindow.toLong(),
+            config.timeWindowUnits,
+        )
```

Comment:
Updated the test to reflect how we use the TimeWindow and kept the value of 1 in the config (see next [change](https://github.com/duckduckgo/Android/pull/2883/files#diff-b77c1cd37a9b9410ea0eeba0bdfe20f083622a308ca8deb780ccc440f0802b20R371)).